devices/baseEstimatorV1/app/robots/iCubGenova04/wholebodydynamics-external.xml

```diff
@@ -0,0 +1,81 @@
+<!--
```

Comment:
Probably you don't need this file because on the real platform `wholebodydynamics` runs independently from the base estimation device

Reply:
Addressed in 82d978f1892d49beb5bdc29df0f343a153b2678a